content/case_histories/mt_isa/processing.rst

```diff
@@ -19,7 +19,7 @@ The ultimate goal is to generate a 3D subsurface conductivity model. However, we
 Data Quality Control
 ********************
 
-As presented in the :ref:`previous<mt_isa_data>` section, the MIMDAS system collects simultaneously a pole-dipole (P-DP) and a dipole-pole (DP-P) data configuration. Accordingly, the P-DP and DP-P data were inverted separately in 2D. The uncertainties are assigned as 5% of the data amplitude with a minimum floor value of 0.02mV. The data are inverted with no noticably bad data points and with the correct normalizations (the data must be normalized by current). The figure :ref:`below <MIM_DC2D_Inv_QC>` shows the observed, predicted, and recovered models for both configurations and for each of the ten 2D lines. While smoothly varying, both the P-DP and DP-P configuration seem to agree on the general distribution of conductivities.
+As presented in the :ref:`previous<mt_isa_data>` section, the MIMDAS system collects simultaneously a pole-dipole (P-DP) and a dipole-pole (DP-P) data configuration. Accordingly, the P-DP and DP-P data were inverted separately in 2D. The uncertainties are assigned as 5% of the data amplitude with a minimum floor value of 0.02mV. The data were inverted and no noticably bad data points were obvious in the data misfit maps. This also means that data were correctly normalized so they corresponded to a unit amplitude current in the transmitter. The figure :ref:`below <MIM_DC2D_Inv_QC>` shows the observed and  predicted data and recovered models for both configurations and for each of the ten 2D lines. There are some regions where there are significant differences in the conductivities obtained from the P-DP and DP-P configurations.  Some of this might be attributable to the fact that the two surveys illuminate buried conductors quite differently (ref ?link to }. Overall, however the inversion results have the same general distribution of conductivity and there doesn't appear to be major issues with normalizations or bad electrodes. 
```

Comment:
there is a link here that needs to be added? @dougoldenburg : what did you have in mind?